apps/website/src/pages/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
```

Comment:
Can we setup a documentation preview in the ci so that when the PR is created, the test site can be viewed via netlify/vercel?

Reply:
@thedoublejay thoughts? idk if this would be too much at this point

Reply:
Create an issue for now, I need to set it up on Vercel itself.

Reply:
I think the Vercel deployment is failing

Reply:
Fixed!